content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -187,3 +187,12 @@ Here are some examples of device plugin implementations:
 
 {{% /capture %}}
 
+{{% capture whatsnext %}}
+
+* Learn about [scheduling GPU resources](/docs/tasks/manage-gpus/scheduling-gpus/) using device plugins
```

Comment:
line 9: The feature state label is rendering at the bottom of the page. Put into body of page?
line 18: High-performance NICs OR high-performance NICs?
line 49, `to advertise 2 `hardware-vendor.example/foo`? Advertising the device references, seems confusing?
line 174, Have a second look at the preview. There appears to be a heading issue toward the lower part of the page, Examples of implementations

Reply:
Thanks for the feedback @kbhawkey!

Reply:
Resolved (I think)